### PR TITLE
Fixed bug - CB-Live-Response

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25152

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/Carbon_Black_Enterprise_Live_Response/Integrations/integration-Carbon_Black_Enterprise_Live_Response.yml [hunk 1]
```diff
@@ -2718,6 +2720,3 @@ script:
       type: boolean
     description: Executes an endpoint memory dump.
   runonce: false
-tests:
```
Comment (Itay4) [inline Packs/Carbon_Black_Enterprise_Live_Response/Integrations/integration-Carbon_Black_Enterprise_Live_Response.yml:2721]: revert this change

File: Packs/Carbon_Black_Enterprise_Live_Response/pack_metadata.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-  "categories": [
-    "Endpoint"
-  ],
-  "tags": [],
-  "useCases": [],
-  "keywords": []
+    "name": "Carbon Black Enterprise Live Response",
+    "description": "Collect information and take action on remote endpoints in real time with Carbon Black Enterprise Live Response.",
+    "support": "xsoar",
+    "currentVersion": "1.1.0",
```
Comment (Itay4) [inline Packs/Carbon_Black_Enterprise_Live_Response/pack_metadata.json:5]:
```suggestion
    "currentVersion": "1.0.1",
```

**Comment by Itay4** [comment]
please remove this line from the code:
`//throw JSON.stringify(fileInfo,null,'\t');`

**Comment by kirbles19** [comment]
@Shellyber Done.

**Comment by Itay4** [comment]
build fails on pack installation, not related to changes done in this PR
merging
